packages/x-utils/src/get-safe-property-chain.ts

```diff
@@ -7,7 +9,10 @@
  * @returns The value of the chain of properties, or `undefined` if any of the intermediate values
  * is not defined.
  */
-export function getSafePropertyChain<T = any>(obj: any, propertyChain: string): T | undefined;
+export function getSafePropertyChain<SomeObject, Path extends ExtractPath<SomeObject>>(
+  obj: SomeObject,
+  propertyChain: Path
```

Comment:
Isn't this missing the `defaultReturn` parameter? As now we have the nullish coalescing, I would prefer to remove it from all signatures.

Reply:
yes, I will do it when this PR is merged

https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions

Reply:
👍🏻